src/index.js

```diff
@@ -125,4 +125,4 @@ setTimeout(() => {
   startSync();
   startAPI();
   openBrowser();
-}, 5000);
+}, 0);
```

Comment:
We delay this because qtumd might have to restart itself if the chain state is corrupted. Is this necessary to remove it?

Reply:
oh, i c. I will change it back to 5000.